integration_tests/correct_forwarder_snapshot.json

```diff
@@ -518,7 +518,7 @@
           ]
         },
         "Layers": [
-          "arn:aws:lambda:sa-east-1:464622532012:layer:Datadog-Node10-x:49"
```

Comment:
Minor thing: Do you need to remove the `Node10` part of the layer ARN here? That should be stable I think. The only unstable part would be the version number `49` at the end.

Reply:
I fixed it using partial replacement. I dont really understand how it works but it does haha